client/my-sites/site-settings/section-general.jsx

```diff
@@ -26,7 +26,8 @@ module.exports = React.createClass({
 				{ ( config.isEnabled( 'manage/site-settings/delete-site' ) && ! site.jetpack && ! site.is_vip ) ?
 				<DeleteSiteOptions
 					site={ this.props.site }
-					purchases={ this.props.purchases } />
+					sitePurchases={ this.props.sitePurchases }
```

Comment:
Super minor, but changing `purchases` to `sitePurchases` for a site-specific component sounds a little redundant to me.